pkgs/top-level/all-packages.nix

```diff
@@ -15921,6 +15921,7 @@ with pkgs;
     boost173
     boost174
     boost175
+    boost176
```

Comment:
```suggestion
```

Reply:
No, we don't need this. I don't have a preference here. Happy to stay with a gap between 1.75 and 1.77.

Reply:
Ideally most of those versions wouldn't be needed and we could drop a lot of them.